### DISCUSSION TITLE
安卓APP无法登陆专业版

### DISCUSSION THREAD

**Discussion by cqbcqb** [getlantern/lantern-forum-cn#244] [General]
从官网下载更新后没有专业版登陆入口，最新版本无法切换语言，现在免费版流量已经用完了，一直不能登陆专业版，我购买专业版一个多月了，一天没用上，之前的版本是连不上去，现在的最新版本是登陆不上去，我买的两年的，我对APP能否使用两年缺乏信心

**Comment by MaxMKMax** [comment]
@cqbcqb 您好。给您带来的不便，深表歉意。您能给我您的专业帐户的电子邮件地址吗？我帮你直接联系负责人。

**Reply by cqbcqb** [reply]
1130069192@qq.com

**Comment by MaxMKMax** [comment]
@cqbcqb  我们已经解决了语言问题。现在可以更改了，您能帮忙检查一下您的版本是否是最新的吗？您可以到我们的Github下载最新版本或查看我们的官方网站。

**Reply by cqbcqb** [reply]
语音问题更改了，但是还是无法登陆专业版，我花钱有什么用，要不退钱吧

**Reply by MaxMKMax** [reply]
@cqbcqb 您好。请问您是 IOS 还是 Android 用户？今晚我会再次为您跟进。请问您无法登录专业版的实际问题是什么呢？

如果您是受到授权码问题影响的ios用户，我们将通过重新延长您的专业版使用时间来补偿您。

**Reply by cqbcqb** [reply]
安卓用户，手机APP根本没有登陆专业版入口，一个是缴费通道，一个是授权其他设备使用，我想知道我本机APP怎么登陆

**Reply by MaxMKMax** [reply]
@cqbcqb 你好。
步骤1: 您可以进入帐户
步骤2: 然后查找专业版的授权设备。
步骤3: 然后使用授权码进行登录。

授权将发送到您的电子邮件或如果您看到任何弹出窗口。

你也可以看看:
https://lantern.io/zh-Hans/faq

**Reply by cqbcqb** [reply]
我尝试登陆好多次了，授权码一直收不到

**Reply by MaxMKMax** [reply]
@cqbcqb 您好. 我们正在想办法修复了. 不好意思!
